examples/test_serial/test_serial.py

```diff
@@ -0,0 +1,60 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+"""
+Serial connector usage example
+******************************
+
+:module: test_serial
+
+:synopsis: show how to use the cc_serial connector
+
+.. currentmodule:: test_serial
+"""
+
+import logging
+import time
+
+import pykiso
+
+# as usual import your auxiliairies
+from pykiso.auxiliaries import com_aux
+
+
+@pykiso.define_test_parameters(
+    suite_id=1,
+    case_id=1,
+    aux_list=[com_aux],
+)
+class TestSerialLoopback(pykiso.BasicTest):
+    """In this test we try to receive the same message which we have send.
+    RX is connected to TX on a serial dongle.
+    """
+
+    def test_run(self):
+        """Executing serial loopback example, let's try to send
+        a message and receive it.
+        """
+        logging.info(
+            f"--------------- RUN: {self.test_suite_id}, {self.test_case_id} ---------------"
```

Comment:
That is not needed right?

Reply:
removed.. added info messages which suites more to the serial test